### DISCUSSION TITLE
Remote Config real-time Support

### DISCUSSION THREAD

**Discussion by sjmorton123** [firebase/firebase-js-sdk#8148] [General]
I have React and React-Native applications that are using Remote Config. Google recently released a real-time aspect of this feature to allow changes to Remote Config to get pushed to devices in real time. The React Native SDK for Remote Config supports a callback event that I can listen for and receive updates in real time. It appears that the firebase-js-sdk library does not support this. Are there plans at some point to allow web projects using Remote Config to hook into the realtime eventing the same way that it is supported on mobile?

<img width="858" alt="image" src="https://github.com/firebase/firebase-js-sdk/assets/25855484/5c8e1af7-7944-4ce6-8306-4143ca7591d9">

**Comment by victordidenko** [comment]
Is there a reason, why this feature still unavailable in JS SDK? I've tried manually craft request to `firebaseremoteconfigrealtime.googleapis.com` and looks like it is working fine 🤔 And on first glance I don't see any reason this could not be implemented in browser with long polling...
